set.mm

```diff
@@ -73922,6 +73936,17 @@ Relations and functions (cont.)
       ( cdm cvv fndmd dmexd eqeltrrd ) ACGBHABCFIACDEJK $.
   $}
 
+  $( The domain of a function is a set iff the function is a set.  (Contributed
+     by AV, 8-Aug-2024.) $)
+  fndmexb $p |- ( F Fn A -> ( A e. _V <-> F e. _V ) ) $=
+    ( wfn cvv wcel fnex wa wceq fndm eqcomd adantr dmexg adantl eqeltrd impbida
+    cdm ) BACZADEBDEZADBFQRGABPZDQASHRQSAABIJKRSDEQBDLMNO $.
+
+  $( The domain of a function is a set iff the function is a set.  (Contributed
+     by AV, 8-Aug-2024.) $)
+  fdmexb $p |- ( F : A --> B -> ( A e. _V <-> F e. _V ) ) $=
+    ( wf wfn cvv wcel wb ffn fndmexb syl ) ABCDCAEAFGCFGHABCIACJK $.
```

Comment:
For both theorems above:
Probably in some of the theorems that follow, you use only the direction that does not require replacement. If this is the case, then state separately that implication, or even inline it in proofs.

Reply:
> For both theorems above: Probably in some of the theorems that follow, you use only the direction that does not require replacement. If this is the case, then state separately that implication, or even inline it in proofs.

For ~fndmexb, we already have both implications ~fnex and ~fndmexd : Using ~fndmexd, the proof of ~fndmexb can be shortened (I will do that).
For ~fdmexb, we only have one implication ~fex. The other implication would be needed for ~mapfset and ~mapfoss - here ~fdmexb is currently used.

Reply:
Ups, there is the other implication for ~fdmexb: ~dmfex (the labeling is not consistent: in ~fn_dm_exd and ~dm_f_ex the parts for the domain and the function are reversed)

Reply:
With commit https://github.com/metamath/set.mm/pull/4153/commits/093c831bfea8970b5fb6085940a3246a0485bac7, the proofs of ~mapfset and ~mapfoss are shortened, using ~dmfex instead of ~fdmexb. As side effect, the dependency on ~ax-rep was removed (also for some proofs using these two theorems).